website/docs/docs/cloud/connect-data-platform/connnect-bigquery.md

```diff
@@ -52,6 +52,122 @@ As an end user, if your organization has set up BigQuery OAuth, you can link a p
 
 To learn how to optimize performance with data platform-specific configurations in dbt Cloud, refer to [BigQuery-specific configuration](/reference/resource-configs/bigquery-configs).
 
+### Optional configurations
+
+In BigQuery, optional configurations let you tailor settings for tasks such as query priority, dataset location, job timeout, and more. These options give you greater control over how BigQuery functions behind the scenes to meet your requirements.
+
+To customize your optional configurations in dbt Cloud:
+
+- Click your name at the bottom left-hand side bar menu in dbt Cloud
+- Select **Profile** from the menu
+- From there, click **Project** and select your BigQuery project
+- Select your BigQuery project from the left-hand menu
+- Click **Settings** towards the top right and a **Project details** window will open
+- Scroll down to **Development Connection** and select BigQuery
+- Click **Edit** and then scroll down to **Optional settings**
+
+<Lightbox src="/img/bigquery/bigquery-optional-config.png" width="70%" title="BigQuery optional configuration"/>
+
+The following are the optional configs you can set in dbt Cloud:
+
+| Config                         | Information                                                                                                                  | Type    | Example                     |
+|--------------------------------|------------------------------------------------------------------------------------------------------------------------------|---------|-----------------------------|
+| Priority                       | Sets the priority for BigQuery jobs (either immediate or queued for batch processing)                                       | String  | `batch` or `interactive`    |
```

Comment:
the information column says 'immediate or queued for batch' but the example is interactive? is immediate a typo? 

i suggest double checking the info column with each description to make sure it's all accurate.